packages/manager/modules/pci/src/projects/project/additional-ips/failover-ips/edit/edit.routing.js

```diff
@@ -12,6 +12,10 @@ export default /* @ngInject */ ($stateProvider) => {
       resolve: {
         breadcrumb: /* @ngInject */ () => null,
         goBack: /* @ngInject */ ($state) => () => $state.go('^'),
+        onCancelClick: /* @ngInject */ (trackClick, goBack) => () => {
+          trackClick('failover-ips::edit::cancel');
```

Comment:
Question: Don't we need prefix (PublicCloud::pci::projects::project::failover-ips) over here like others or its just like this? We are using `failover-ips:::` in some more files as well, can we create it as constant and consume?

Reply:
The common prefix `PublicCloud::pci::projects::project::additional-ips` is maintained in the `trackClick` function.